lessons/Early-Math/associative-property/langs/en-us/pages/discover-assoc.adoc

```diff
@@ -0,0 +1,66 @@
+= Discover the Associative Property
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Take a look at the two Circles of Evaluation below. Now, evaluate them to help you decide if the Associative Property holds for problems involving *addition*.
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(+ (+ 10 8) 3))} | @show{(coe  '(+ 10 (+ 8 3)))}
```

Comment:
I can live with this workbook page, but I think it would be even more powerful if it had 3 columns (thereby showing all grouping possibilities)

eg

17 + 4 + 23

(17 + 4) + 23 ...  17 + (4 + 23) ...   (17 + 23) + 4